src/main/kotlin/net/maizegenetics/phgv2/api/ExportHaplotypeGraph.kt

```diff
@@ -31,7 +37,9 @@ private val myLogger = LogManager.getLogger("net.maizegenetics.phgv2.api.ExportH
 fun exportMultiSampleHVCF(
     graph: HaplotypeGraph,
     filename: String,
-    referenceGenome: String? = null
+    referenceGenome: String? = null,
+    symbolicAllele: SymbolicAllele = SymbolicAllele.CHECKSUM,
+    rangeBedfile: String? = null
```

Comment:
Add a comment in the file header that explains why the rangeBedfile is needed .  A short paragraph about the refRange numbers/ids and how they change if you have the full list of ref ranges vs just those in the graph.  It could be helpful to people looking at this file later.

Reply:
Ok, I added this